### DISCUSSION TITLE
If i dot remember the path admin base URL?

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#1734] [Questions & Answers] [answered]
Basically, I always change the admin base URL path.

After a long time I went to check my old test installation of OpenMage but I don't remember the path I put.

I search the net for answers and find suggestions indicating to check the local.xml file in app / etc / but I don't see a custom path.

I thought about comparing: I searched the database of another installation that using the base url path of the pah admin that I know, but couldn't find anything.

Now the question is serious.

How can I find the custom route if I forget or lose it?

**Comment by addison74** [comment]
There is no record in DB for the custom Backend path. You will find it here **/app/etc/local.xml**

```
    <admin>
        <routers>
            <adminhtml>
                <args>
                    <frontName><![CDATA[my_custom_path]]></frontName>
                </args>
            </adminhtml>
        </routers>
    </admin>
```

If you don't remember it or if there isn't one created in the file just use admin or you can create a new one. There is no need to re-install Magento as I saw as a bad solution in the past in a few posts.

**Reply by Danieleeffe1** [reply]
Thanks for the reply

I had already tried as you suggested but it returns me 404 page.

I also tried to create a new path by replacing "my_custom_path" with "admin123" but nothing ...

to be honest I don't even remember if it was openmage or magento 1.9.4.5 anymore

**Comment by addison74** [comment]
In that file is the only place where you can change the path to the Backend. You can also do a search with Google as it is full of resources on this topic.

Try clearing the browser cache if it's Google Chrome. Also restart the webserver service (Apache, Nginx, ...).

If you do not get positive results then recreate the installation as it is in a test environment, just the file system part, without importing the database. Delete the /app/etc/local.xml file and load the frontend. If the installation process went normally you should have a new local.xml file and a confirmation that at least on the side of the files you are using it would be complete.

**Comment by addison74** [comment]
After installation this is the default part for Backend path of local.xml file

```
    <admin>
        <routers>
            <adminhtml>
                <args>
                    <frontName><![CDATA[admin]]></frontName>
                </args>
            </adminhtml>
        </routers>
    </admin>
```

**Comment by justinbeaty** [comment, marked as answer]
@Danieleeffe1 You may want to check in the database here:

```
select * from core_config_data where path = 'admin/url/custom_path';
```

@ADDISON74 The custom path would only be in local.xml if set during installation. If you set through the admin panel later it will be in the database.

**Comment by addison74** [comment]
@justinbeaty - you are right. The custom admin url/path can be set up in Backend: System -> Configuration -> Advanced -> Admin.

![Screenshot 2021-07-15 005154](https://user-images.githubusercontent.com/8360474/125697937-b6502d4b-0b50-494f-b920-3189df1e11c1.jpg)
